osu.Game/Overlays/BeatmapSetOverlay.cs

```diff
@@ -114,10 +115,11 @@ private void updateScores(BeatmapInfo beatmap)
         }
 
         [BackgroundDependencyLoader]
-        private void load(APIAccess api, RulesetStore rulesets)
+        private void load(APIAccess api, RulesetStore rulesets, BeatmapManager manager)
         {
             this.api = api;
             this.rulesets = rulesets;
+            this.manager = manager;

```

Comment:
again, this doesn't handle the case where the beatmap is added. it could have been dragged in; it could have been downloaded in the background.

I also see no reason this is done here and not in the header or the button instead.